src/globus_sdk/experimental/globus_app/globus_app.py

```diff
@@ -198,12 +187,19 @@ def __init__(
         # our initial scope requirements
         self._validating_token_storage = ValidatingTokenStorage(
             token_storage=self._token_storage,
-            scope_requirements=self.scope_requirements,
+            scope_requirements=self._scope_requirements,
         )
 
         # initialize our authorizer factory
         self._initialize_authorizer_factory()
 
+        # create a consent client for token validation
+        # reducing the scope requirements to barebones openid (user identification)
+        # additionally, this will ensure that openid scope requirement is always
+        # registered (it's required for token identity validation).
+        consent_client = AuthClient(app=self, app_scopes=[Scope(AuthScopes.openid)])
+        self._validating_token_storage.set_consent_client(consent_client)
```

Comment:
Nope, the client creation registers openid automatically. We don't do any deduping, so with the old block, openid would be registered twice.

Demo:
```python
>>> from globus_sdk.experimental.globus_app import UserApp
>>>
>>> app = UserApp("my-cool-app", client_id="61338d24-54d5-408f-a10d-66c06b59f6d2")
>>> app.get_scope_requirements("auth.globus.org")
(Scope('openid'),)
```

Reply:
Oh, excellent! That was my main concern. I need to reread and see where or how that happens, and evaluate the PR in that light.

Reply:
It happens because of the `app_scopes` kwarg in the code section you highlighted above.

Code ref: [client.py#L157-L162](https://github.com/globus/globus-sdk-python/blob/main/src/globus_sdk/client.py#L157-L162)

Reply:
I was confused and reread everything, but I finally understood when I realized that this client is created with `app=self`. Somehow I missed that -- now I get it.

Marking this thread resolved.